### PR TITLE
Notify admins when someone activates a smoke machine

### LINKED ISSUE #31578: Smoke Machine does not produce an admin message upon activation
Smoke grenades and smoke reactions both do. It makes it much much easier to deal with misuse of this feature as it is very easy to abuse, and the smoke machine literally makes it shit-your-diaper baby tier to mess around with smoke.

For ease of use and consistency's sake please add an admin message similar to other smoke reactions when the machine is activated. Thankyou.

### PR DESCRIPTION
**Pull request by feemjmeem**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Fixes #31578

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm [hunk 1]
```diff
@@ -106,6 +106,7 @@
 		if("power")
 			on = !on
 			if(on)
+				message_admins("[key_name_admin(usr)] activated a smoke machine that contains [english_list(reagents.reagent_list)] at [ADMIN_COORDJMP(src)].")
 				log_admin("[key_name(usr)] activated a smoke machine that contains [english_list(reagents.reagent_list)] at [COORD(src)].")
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/machinery/smoke_machine.dm]: Unrelated to this PR, but how many fucking log_admin/message admin pairs have this exact fucking format. @optimumtact @nfreader

**Review by Cyberboss — APPROVED** [review]

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (feemjmeem) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: all of them; it's almost like we should make a handler for it

**Comment by sawrge** [comment]
Domo arigato.

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (nfreader) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: Logging needs a serious overhaul, but that's a topic for another time

Comment (optimumtact) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: Please do not ping me for pointless reasons

**Comment by optimumtact** [comment]
your changelog entry is missing an opening :cl:

**Comment by feemjmeem** [comment]
fixed

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (feemjmeem) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: @optimumtact

**Pull request merged**